app/src/main/java/pl/szczodrzynski/edziennik/data/api/edziennik/vulcan/data/hebe/VulcanHebeMessages.kt

```diff
@@ -66,86 +66,95 @@ class VulcanHebeMessages(
         }
         apiGetList(
             TAG,
-            VULCAN_HEBE_ENDPOINT_MESSAGES,
-            HebeFilterType.BY_PERSON,
-            folder = folder,
+            VULCAN_HEBE_ENDPOINT_MESSAGEBOXES,
```

Comment:
Do not download message boxes every time. Add this as a separate endpoint that will sync i.e. every week, and store the message box ID in profile data.